pkgs/applications/science/misc/snakemake/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, python3Packages }:
```

Comment:
Does the author of snakemake say this only works on python3? It seems like you could specify python3 in the `top-level/all-packages.nix` instead of making this derivation specific.

Reply:
Yes, Snakemake [requires python >= 3.5](https://bitbucket.org/snakemake/snakemake/src/4e41054b509aa0a0913c708e859b50833aa08769/setup.py?at=v5.2.2&fileviewer=file-view-default#setup.py-12).
However, it's makes sense set the python version at `top-level/all-packages.nix`. Done.